### PR TITLE
Minimize pseudo elements

### PR DESCRIPTION
**Pull request by stevenbfine**
Remove redundant pseudo element style properties.

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -135,6 +145,66 @@ function minimizeStyles(message) {
...
+  var element = doc.getElementById(id);
+  if (element) {
+    var originalStyleMap = message.pseudoElementSelectorToCSSMap[selector];
+    var requiredStyleMap = {};
+    // We compare the computed style before and after removing the pseudo element
+    // and accumulate the differences in |requiredStyleMap|. Because some
+    // properties affect other properties, such as border-style: solid causing a
+    // change in border-width, we do this iteratively until a fixed-point is
+    // reached (or |maxNumberOfIterations| is hit).
+    for (var i = 0; i < maxNumberOfIterations; i++) {
```
Comment (progers) [inline popup.js:178]: Can you add a TODO here to unify this logic with minimizeStyles?
// TODO(sfine): Unify this logic with minimizeStyles.

(it's okay if you never finish the TODO, it's a message to future readers of the code)

File: popup.js [hunk 2]
```diff
@@ -135,6 +145,66 @@ function minimizeStyles(message) {
...
+    finalPseudoElements) {
+  var maxNumberOfIterations = 5;
+  var match = selector.match(/^#(.*):(:.*)$/);
+  var id = match[1];
+  var type = match[2];
+  var element = doc.getElementById(id);
+  if (element) {
+    var originalStyleMap = message.pseudoElementSelectorToCSSMap[selector];
+    var requiredStyleMap = {};
+    // We compare the computed style before and after removing the pseudo element
```
Comment (progers) [inline popup.js]: Can you mention how the pseudo element is removed? I.e., that you remove it by removing the id.

File: popup.js (hunk 1 above)
Comment (stevenbfine) [reply popup.js:178]: Done

File: popup.js (hunk 2 above)
Comment (stevenbfine) [reply popup.js]: Done

Comment (stevenbfine) [inline popup.js]: Done

**Comment by progers** [comment]
LGTM

**Pull request merged**